src/arviz_plots/plots/rank_dist_plot.py

```diff
@@ -0,0 +1,439 @@
+    The following examples focus on behaviour specific to ``plot_rank_dist``.
+    For a general introduction to batteries-included functions like this one and common
+    usage examples see :ref:`plots_intro`
+
+    Default plot_rank_dist (``compact=True`` and ``combined=False``). In this case,
+    the multiple coordinate values are overlaid on the same plot for multidimensional values;
+    by default, the color is mapped to all dimensions of each variable (but `sample_dims`)
+    to allow distinguishing the different coordinate values.
+
+    As ``combined=False`` each chain is also being plotted, overlaying them on their
+    corresponding plots; as the color property is already taken, the chain information
+    is encoded in the linestyle as default.
+
+    Both mappings are applied to the rank and dist elements.
+
+    .. plot::
+        :context: close-figs
+
+        >>> from arviz_plots import plot_rank_dist, style
+        >>> style.use("arviz-variat")
+        >>> from arviz_base import load_arviz_data
+        >>> centered = load_arviz_data('centered_eight')
+        >>> coords = {"school": ["Choate", "Deerfield", "Hotchkiss"]}
+        >>> pc = plot_rank_dist(centered, coords=coords, compact=True, combined=False)
+        >>> pc.add_legend("school")
+
+    plot_rank_dist with ``compact=True`` and ``combined=True``. The aesthetic mappings
+    stay the same as in the previous case, but now the linestyle property mapping
+    is only taken into account for the rank as in the left column, we use
+    the data from all chains to generate a single distribution representation
+    for each variable+coordinate value combination.
+
+    Similarly to the first case, this default and now only mapping is applied to both
+    the rank and the dist elements.
+
+    .. plot::
+        :context: close-figs
+
+        >>> pc = plot_rank_dist(centered, coords=coords, compact=True, combined=True)
+        >>> pc.add_legend("school")
+
+    When ``compact=False``, each variable and coordinate value gets its own plot,
+    and so the color property is no longer used to encode this information.
+    Instead, it is now used to encode the chain information.
+
+    .. plot::
+        :context: close-figs
+
+        >>> pc = plot_rank_dist(centered, coords=coords, compact=False, combined=False)
+
+    Similarly to the other ``combined=True`` case, the aesthetics stay the same
+    as with ``combined=False``, but they are ignored by default when plotting
+    on the left column.
+
+    .. plot::
+        :context: close-figs
+
+        >>> pc = plot_rank_dist(centered, coords=coords, compact=False, combined=True)
+        >>> pc.add_legend("chain")
+
+    References
+    ----------
+    .. [1] Säilynoja T, Bürkner PC. and Vehtari A. *Graphical test for discrete uniformity and
+    its applications in goodness-of-fit evaluation and multiple sample comparison*.
+    Statistics and Computing 32(32). (2022) https://doi.org/10.1007/s11222-022-10090-6
```

Comment:
```suggestion
    .. [1] Säilynoja T, Bürkner PC. and Vehtari A. *Graphical test for discrete uniformity and
       its applications in goodness-of-fit evaluation and multiple sample comparison*.
       Statistics and Computing 32(32). (2022) https://doi.org/10.1007/s11222-022-10090-6
```

this should fix the warnings and get the citation to render correctly